app/src/main/java/com/example/dynamicdiet/dto/Entries.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.dynamicdiet.dto
+
+import java.util.*
+//renamed weight to weightLbs due to it being in pounds
+//renamed Weight to Entries
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.